js/components/admin-page-settings.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * WordPress dependencies.
+ */
+const { __ } = wp.i18n;
+const { PluginDocumentSettingPanel } = wp.editPost;
+const { PanelRow } = wp.components;
+const { compose } = wp.compose;
+const { withSelect, withDispatch } = wp.data;
+
+const icon = 'editor-ul';
+
+import MetaTextControl from './meta-text-control';
+import MetaSelectControl from './meta-select-control';
+import MetaToggleControl from './meta-toggle-control';
```

Comment:
If we create an `index.js` file, like we do with Hello Charts ([example](https://github.com/hello-wp/hello-charts/blob/main/js/common/components/index.js)), we could simplify this to:

```suggestion
import { MetaTextControl, MetaSelectControl, MetaToggleControl } from '.';
```